go/ms/internal/sqlite/db.go

```diff
@@ -0,0 +1,226 @@
+// Copyright 2018 ETH Zurich
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package sqlite
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"sync"
+
+	_ "github.com/mattn/go-sqlite3"
+
+	"github.com/scionproto/scion/go/lib/ctrl"
+	"github.com/scionproto/scion/go/lib/infra/modules/db"
+	"github.com/scionproto/scion/go/lib/serrors"
+	"github.com/scionproto/scion/go/proto"
+)
+
+var Db *DB
```

Comment:
I don't see this variable ever used.
Also, where are you making sure that the DB gets closed on exit?

Reply:
The variable is used in other files that are not in this commit. 
From what I see DB is created by doing db.NewSqlite() which then does Open using database/sql. And the Open creates a pool of connections (https://golang.org/pkg/database/sql/#Open) so does not need to be closed explicitly and all connections are closed when the program exits. Am I missing something?